notebook/static/notebook/js/notebook.js

```diff
@@ -113,7 +113,7 @@ define(function (require) {
                         }
                     }, function (err) {
                         console.log("No CodeMirror mode: " + lang);
-                        callback(err, code);
+                        callback(null, code);
```

Comment:
It might be worth logging the error, if not raising it, just in case the error is caused by something else. From the looks of the code, it seems like the first condition above (`if (!mode)`) is meant to be the one handling a missing mode, but it is now triggering the require error path. I'm not quite sure what changed, but I'm guessing one or the other is never being called.